hr_cae_contract/views/hr_contract.xml

```diff
@@ -21,6 +21,11 @@
                         type="object"
                         attrs="{'invisible': [('has_termination_contract', '=', True)]}"
                         class="oe_read_only"/>
+                <button name="create_document"
```

Comment:
shouldn't you have the invisible domain depend on state rather than on `has_termination_contract`?

Reply:
Absolutely! Ill do that next.